src/types/Client.d.ts

```diff
@@ -25,4 +27,5 @@ export default class Client {
   query<T = object>(expr: ExprArg, options?: QueryOptions): Promise<T>
   paginate(expr: Expr, params?: object, options?: QueryOptions): PageHelper
   ping(scope?: string, timeout?: number): Promise<string>
+  stream(expr: Expr, options?: object): Subscription
```

Comment:
Could you also type this `options` argument? It has a single field called `fields` and it is a list of strings.